copper-server-api/src/main/java/com/pogeyan/cmis/api/utils/Helpers.java

```diff
@@ -404,4 +404,16 @@ public static String splitFilterQuery(String filter) {
 		result.add(PropertyIds.BASE_TYPE_ID);
 		return result;
 	}
+
+	public static String[] getTypeMappedColumns(List<String> fieldAccess) {
+		String[] defaultProperty = new String[] { "_id", "localName", "localNamespace", "queryName", "displayName",
```

Comment:
Move default property list into helper class, we are using it in several places.